bigchaindb/web/views/transactions.py

```diff
@@ -98,7 +98,7 @@ def post(self):
 
         try:
             tx_obj = Transaction.from_dict(tx)
-        except (InvalidHash, InvalidSignature):
+        except (ValidationError, InvalidSignature):
             return make_error(400, 'Invalid transaction')
```

Comment:
To expand on this, the exception data is huge and messy and im not sure it's appropriate to pass it to a client as-is, maybe it's not the highest priority to solve it right now.